tiledbcontents/tiledbcontents.py

```diff
@@ -490,6 +491,8 @@ def _write_bytes_to_array(
             if type is not None:
                 A.meta["type"] = type
 
+        self.cached_contents = {"contents": contents}
```

Comment:
I believe we should set this on the Array class not the `TileDBContents` class? Else we don't know what array it is for?

Reply:
Thanks for this